Packs/SymantecEDR/IncidentFields/incidentfield-Symantec_EDR_Recommended_Action.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "incident_recommendedaction",
```

Comment:
Use the same convention as other pack specific fields: "incident_edrrecommendedaction"